config/manager/manager.yaml

```diff
@@ -146,7 +146,7 @@ spec:
           resources:
             limits:
               cpu: 500m
-              memory: 1Gi
+              memory: 2.5Gi
```

Comment:
looks like we are mixing Mi and Gi in various configs. Can we stick to one unit?